File: Python/ChatGPT4o/Decorator/round1/SecuritySystem.py
from CarDecorator import CarDecorator

class SecuritySystem(CarDecorator):
    """Decorator class for adding a security system to a car."""

    def description(self):
        """Adds a security system to the car description."""
        return self._car.description() + ", Security System"

    def cost(self):
        """Adds the cost of a security system to the car."""
        return self._car.cost() + 2000

File: app/src/test/java/sqa_project/GithubCopilot/decorator/round1/SecuritySystem.java
package sqa_project.GithubCopilot.decorator.round1;

public class SecuritySystem extends CarDecorator {
    public SecuritySystem(Car car) {
        super(car);
    }

    @Override
    public String getDescription() {
        return decoratedCar.getDescription() + ", Security System";
    }

    @Override
    public double getCost() {
        return decoratedCar.getCost() + 2500;
    }
}